src/main/resources/avro/sequenceAnnotations.avdl

```diff
@@ -98,9 +103,9 @@ protocol SequenceAnnotations {
     }> attributes;
 
     /**
-    Identifiers of children nodes
+    Identifier for the parent annotation, if any.
     */
-    array<string> childrenIds;
+    string parentId;
```

Comment:
Agreed.

On Thu, Feb 5, 2015 at 8:11 AM, Mark Diekhans <notifications@github.com>
wrote:

> In src/main/resources/avro/sequenceAnnotations.avdl
> <https://github.com/diekhans/ga4gh-schemas-genome-ann/pull/16#discussion_r24148422>
> :
>
> >      */
> > -    array<string> childrenIds;
> > +    string parentId;
>
> OK, I committed multiple parent ids. I can think of nothing cleaner. I
> also took a crack at a better representation of the attributes. Allowing
> for multiple values per attribute name. The current GFF approach leads to a
> bunch of ad-hoc parsing of string attributes when complex or multiple
> values are required. I like the concept, but not completely crazy about the
> implantation because avro isn't very expressive. I like CH's suggestion of
> featureType as the attribute and I think this leads to SeqAnnFeature as the
> record. I could be total open to dropping the SeqAnn prefix too, that could
> better be handled by namespaces.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/diekhans/ga4gh-schemas-genome-ann/pull/16/files#r24148422>
> .
>